pkg/client/postgres/tasks.go

```diff
@@ -95,9 +98,40 @@ func (db *PostgresDB) ScheduleBackup(schedule string) error {
 	return db.client.Apply(db.Namespace, job)
 }
 
-func (db *PostgresDB) ListBackups() ([]string, error) {
-	var backups []string
-	return backups, nil
+func (db *PostgresDB) ListBackups(quiet bool, num int) error {
```

Comment:
This function should return a slice of strings and an error, the printing to the console should happen in the cmd package

Reply:
can't return just slice of strings but can be a slice of structs since we will need the age, size etc